lib/compat/wordpress-6.0/edit-form-blocks.php

```diff
@@ -8,10 +8,17 @@
 /**
  * Optimizes the preload paths registered in Core (`edit-form-blocks.php`).
  *
- * @param array $preload_paths Preload paths to be filtered.
+ * @param array                   $preload_paths        Preload paths to be filtered.
+ * @param WP_Block_Editor_Context $block_editor_context The current block editor context.
+ *
  * @return array
  */
-function gutenberg_optimize_preload_paths( $preload_paths ) {
+function gutenberg_optimize_preload_paths( $preload_paths, $block_editor_context ) {
+	// optimize only the post editor page, not site or widgets.
+	if ( 'core/edit-post' !== $block_editor_context->name ) {
```

Comment:
```suggestion
	if ( isset( $block_editor_context->name ) && 'core/edit-post' !== $block_editor_context->name ) {
```

The `name` property doesn't exist in WP >= 5.9, need to include a check to avoid PHP notice.

Reply:
isn't the min requirement 5.9 now?

Reply:
It's WP 6.0, so I don't think we need these changes anymore. See my previous comment - https://github.com/WordPress/gutenberg/pull/40587#issuecomment-1348248852.